docs/css/react.scss

```diff
@@ -260,6 +260,22 @@ h1, h2, h3, h4, h5, h6 {
     &.active {
       color: $primary;
     }
+
+    &[href*="http"] {
```

Comment:
Can we just add a class to these links instead? https://github.com/facebook/react/blob/de09e0acd82d2de955af4c395fd9066292adef70/docs/_plugins/sidebar_item.rb#L6-L7